brave/brave-core
# Ads Info UI

Fixes: https://github.com/brave/brave-browser/issues/3468
Requires: https://github.com/brave-intl/bat-native-confirmations/pull/69

This ads estimated earnings and total notifications received to the Ads box in Rewards settings.

<img width="641" alt="screen shot 2019-02-23 at 3 33 59 am" src="https://user-images.githubusercontent.com/8732757/53285262-e361e880-371b-11e9-8fcd-405e6f5a479b.png">

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Start Brave from a fresh profile, enable Rewards/Ads
2. Go visit a webpage (such as imdb.com), idle for 15 seconds, then move to receive a notification
3. Go to `brave://rewards` and confirm that the `notifications received` is now 1.
4. Repeat step 2 but view the ads. Confirm the count has incremented appropriately.

Note: Once ads with BAT are put in to the catalog, estimated earnings can be confirmed.

## Reviewer Checklist:
